### PR TITLE
[SBOM] Send usage metrics

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Send 2 metrics:
- datadog.agent.sbom.host.running when SBOM generation for host is enabled
- datadog.agent.sbom.container_images.running when SBOM generation for container images is enabled

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by 0intro — APPROVED** [review]

File: pkg/collector/corechecks/sbom/check.go [hunk 1]
```diff
@@ -173,13 +180,23 @@ func (c *Check) Run() error {
 			c.processor.processContainerImagesRefresh(c.workloadmetaStore.ListImages())
 		case <-hostPeriodicRefreshTicker.C:
 			c.processor.processHostRefresh()
+		case <-metricTicker.C:
+			c.sendUsageMetrics()
```
Comment (vickenty) [inline pkg/collector/corechecks/sbom/check.go:184]: Would it be possible to send metrics from the `Run` method?

File: pkg/collector/corechecks/sbom/check.go [hunk 2]
```diff
@@ -173,13 +180,25 @@ func (c *Check) Run() error {
...
 		case <-c.stopCh:
 			c.processor.stop()
 			return nil
 		}
 	}
 }
 
+func (c *Check) sendUsageMetrics() {
+	c.sender.Count("datadog.agent.sbom.container_images.running", 1.0, "", nil)
```
Comment (vickenty) [inline pkg/collector/corechecks/sbom/check.go:193]: This seems to be a duplicate of `datadog.agent.checks.runs`?

File: pkg/collector/corechecks/sbom/check.go [hunk 3]
```diff
@@ -173,13 +180,25 @@ func (c *Check) Run() error {
...
+
+	if ddConfig.Datadog.GetBool("sbom.host.enabled") {
+		c.sender.Count("datadog.agent.sbom.hosts.running", 1.0, "", nil)
```
Comment (vickenty) [inline pkg/collector/corechecks/sbom/check.go:196]: The value of this metric never changes. Would it make sense to use agent config inventory feature for this?

File: pkg/collector/corechecks/sbom/check.go (hunk 1 above)
Comment (lebauce) [reply pkg/collector/corechecks/sbom/check.go:184]: Do you mean directly calling it from `Run` instead of calling a submethod from `Run`

Comment (vickenty) [reply pkg/collector/corechecks/sbom/check.go:184]: Oops, I missed the fact that this is already inside Run, and a long-running check to boot. Never mind then.

**Review by vickenty — APPROVED** [review]

File: pkg/collector/corechecks/sbom/check.go (hunk 2 above)
Comment (lebauce) [reply pkg/collector/corechecks/sbom/check.go:195]: Is this metric tagged with the check ? Anyway, I don't think we can use it as it's a long running check :-/

**Review by jinroh — APPROVED** [review]

**Review by AliDatadog — APPROVED** [review]

**Pull request merged**